### PR TITLE
Status

### PR DESCRIPTION
**Pull request by yangevelyn**
Status page functionality:
 - display nickname, group name
 - display tasks (sorted by completion)
 - add tasks
 - button for task completion

### CODE REVIEW

**Review by labib666 — APPROVED** [review]
LGTM!

File: webapp/status_server.py [hunk 1]
```diff
@@ -8,20 +8,89 @@
...
+
+
+def add_task(group, user):
+  if request.method == "POST":
+    db = webapp.pb.database()
+
+    content = request.form.get('task')
+    uid = user['user_id']
+
+    key = db.child('groups').child(group).child(uid).child('tasks').push({
```
Comment (shaynemei) [inline webapp/status_server.py]: maybe we want to change this to users -> uid -> tasks?
(and other queries for tasks)

File: webapp/status_server.py [hunk 2]
```diff
@@ -8,20 +8,89 @@
...
+      g.user = user
+  except:
+      return redirect(url_for("login"))
+
+  nickname, group = get_info(user)
+  uid = user['user_id']
+  db = webapp.pb.database()
+
+  all_tasks = db.child('groups').child(group).child(uid).child('tasks').get()
+  print(all_tasks.val())
```
Comment (shaynemei) [inline webapp/status_server.py]: let's remove all print statements

File: webapp/auth_server.py [hunk 3]
```diff
@@ -65,27 +64,37 @@ def signup():
...
+          'tasks': {}
         }
       })
     else:
       db.child('groups').update({
         group: {
           id: {
-            'nickname': name
+            'nickname': name,
+            'tasks': {}
```
Comment (shaynemei) [inline webapp/auth_server.py]: Do we need to store tasks under groups?

File: webapp/auth_server.py [hunk 4]
```diff
@@ -65,27 +64,37 @@ def signup():
     return render_template("signup.html")
 
 
-def add_user(id, name, group, join):
+def add_user(email, id, name, group, join):
```
Comment (shaynemei) [inline webapp/auth_server.py:67]: we might want to init resources -> water, food, metal for non-joining users. Maybe we can start with 20 units each?

**Review by shaynemei — CHANGES_REQUESTED** [review]
Left some comments!

**Review by shaynemei — APPROVED** [review]
LGTM!

**Pull request merged**